genfragments/ThirteenPointSixTeV/Higgs/HH/HHto4Vto1L4Jplus_TuneCP5_13p6TeV_madgraph-pythia8.py

```diff
@@ -0,0 +1,41 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8ConcurrentHadronizerFilter",
+                         maxEventsToPrint = cms.untracked.int32(1),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(13600.),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            '23:mMin = 0.05',
+            '24:mMin = 0.05',
+            '25:m0 = 125.0',
+            '25:onMode = off',
+            '25:onIfMatch = 24 -24',
+            '25:onIfMatch = 23 23',
+            '23:onMode = off',
+            '23:onIfAny = 1 2 3 4 5 12 14 16',
```

Comment:
you will have 1L2J4Nu or 1L6J event

Reply:
Yes, that's intended. Although now that I think about it, `HHto4Vto1L4Jplus` is a bit misleading here.. What would you recommend for the name though? Just `HHto4Vto1L`?

Reply:
@menglu21 I've now renamed the fragment.